.github/workflows/release.yaml

```diff
@@ -48,7 +48,9 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-receiver
+          images: |
+            ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-receiver
+            flashbots/tdx-orderflow-proxy-receiver
```

Comment:
I think we want to remove the `tdx` prefix, as this will/can be used for other tees too.

maybe `buildernet-orderflow-proxy`, or just `orderflow-proxy` 🤔